src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -168,6 +176,69 @@ private IEnumerable<CompletionItem> GetResourceTypeCompletions(SemanticModel mod
                 .ToList();
         }
 
+        private IEnumerable<CompletionItem> GetModulePathCompletions(SemanticModel model, BicepCompletionContext context)
+        {
+            if (!context.Kind.HasFlag(BicepCompletionContextKind.ModulePath)
```

Comment:
I think it would be good to break out the first condition into its own check, just to be consistent with other methods and make the code a bit simpler to read:

```csharp
if (!context.Kind.HasFlag(BicepCompletionContextKind.ModulePath))
{
    return Enumerable.Empty<CompletionItem>();
}
```